build.sh

```diff
@@ -175,7 +175,7 @@ if hasArg --cmake_default_generator; then
     CMAKE_GENERATOR_OPTION=""
 fi
 if hasArg --pydevelop; then
-    PYTHON_ARGS_FOR_INSTALL="-m pip install -e ."
+    PYTHON_ARGS_FOR_INSTALL="-m pip install -e --no-build-isolation --no-deps ."
```

Comment:
Looks like `-e` has to be at the end.
```
ERROR: --no-build-isolation is not a valid editable requirement. It should either be a path to a local project or a VCS URL (beginning with bzr+http, bzr+https, bzr+ssh, bzr+sftp, bzr+ftp, bzr+lp, bzr+file, git+http, git+https, git+ssh, git+git, git+file, hg+file, hg+http, hg+https, hg+ssh, hg+static-http, svn+ssh, svn+http, svn+https, svn+svn, svn+file).
```

Reply:
FYI, I made this change in [my PR](https://github.com/rapidsai/cugraph/pull/3065) and tested it locally. We should be able to close this PR now if that's easier.